apps/rush-lib/src/logic/InstallHelpers.ts

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as colors from 'colors';
+import * as os from 'os';
+import * as path from 'path';
+import { LockFile, FileSystem } from '@rushstack/node-core-library';
+
+import { LastInstallFlag } from '../api/LastInstallFlag';
+import { Utilities } from '../utilities/Utilities';
+import { PackageManagerName } from '../api/packageManager/PackageManager';
+import { RushConfiguration } from '../api/RushConfiguration';
+import { RushGlobalFolder } from '../api/RushGlobalFolder';
+
+export class InstallHelpers {
+  /**
+   * If the "(p)npm-local" symlink hasn't been set up yet, this creates it, installing the
+   * specified (P)npm version in the user's home directory if needed.
+   */
+  public static async ensureLocalPackageManager(
```

Comment:
This is just moved from `InstallManager`, right?

Reply:
Yes, the code wasn't changed in any meaningful way.